web/cobrands/fixmystreet/images/chevron-black-right.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="43" height="33"><path fill="#000" d="M0 0h13l13 16-13 16H0l13-16"/></svg>
```

Comment:
Was going to say do you want to swap the Westminster cobrand to use this as well, but I see that's off-black.